src/Framework/Conventions/KinePhaseSpace.h

```diff
@@ -38,6 +38,8 @@ typedef enum EKinePhaseSpace {
   kPSlogyfEx,
   kPSxyfE,
   kPSlogxlogyfE,
+  kPSxQ2fE,
```

Comment:
These should be added at the end to avoid compatibility issues

Reply:
Ok. Easy to fix.